back/app/models/user.rb

```diff
@@ -221,7 +216,9 @@ def to_token_payload
     {
       sub: id,
       roles: compacted_roles,
-      exp: token_lifetime.from_now.to_i
+      exp: token_lifetime.from_now.to_i,
+      cluster: CL2_CLUSTER,
+      tenant: Tenant.current.id
```

Comment:
Interested to know why these are now needed

Reply:
These lines come from one more merged patch, `back/engines/commercial/multi_tenancy/app/models/multi_tenancy/patches/user.rb`